src/platform-includes/sourcemaps/legacy-uploading-methods/javascript.mdx

```diff
@@ -129,7 +129,7 @@ module.exports = {
       org: "___ORG_SLUG___",
       project: "___PROJECT_SLUG___",
 
-      // Auth tokens can be obtained from https://sentry.io/settings/account/api/auth-tokens/
+      // Auth tokens can be obtained from the your User Settings
       // and need `project:releases` and `org:read` scopes
```

Comment:
Whats the reason we are not replacing this with org tokens here?

Reply:
This is in the legacy docs. People here may use a self hosted version that is not yet compatible with org auth tokens.

Reply:
ah, makes sense! 👍